aspnetcore/fundamentals/minimal-apis/includes/parameter-binding8.md

```diff
@@ -90,7 +90,7 @@ Parameters declared in route handlers are treated as required:
 * If a request matches the route, the route handler only runs if all required parameters are provided in the request.
 * Failure to provide all required parameters results in an error.
 
-[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_op1)]
+:::code language="csharp" source="~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs" id="snippet_op1" highlight="4":::
```

Comment:
Added highlights for the optional parameter:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/f13c1a88-44f3-4a99-afeb-111705551826)